frontend/app/mint/[input_index]/page.tsx

```diff
@@ -0,0 +1,361 @@
+"use client"
+import { useSearchParams } from 'next/navigation'
+import {  useState, useEffect, useMemo } from 'react'
+import Image from 'next/image';
+import { Contract, ethers } from 'ethers';
+import { useConnectWallet } from "@web3-onboard/react";
+
+
+import { ReplayScore, getOutputs } from "../../backend-libs/app/lib";
+import { envClient } from '../../utils/clientEnv';
+import { fontPressStart2P } from "../../utils/font"
+import nftAbiFile from "../../contracts/RivesScoreNFT.sol/RivesScoreNFT.json"
+import { delay } from "../../utils/util";
+import CheckIcon from "../../components/svg/CheckIcon";
+import ErrorIcon from "../../components/svg/ErrorIcon";
+import CloseIcon from "@mui/icons-material/Close";
+
+const nftAbi: any = nftAbiFile;
+
+const BASE64_KEY = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'
+export function byteToBase64(bytes: Uint8Array): String {
```

Comment:
This export makes the build fail.
```suggestion
function byteToBase64(bytes: Uint8Array): String {
```